### PR TITLE
CT-746 file prepare async

### PR DESCRIPTION
**Pull request by zajca**
Jira: CT-746
KBC: https://github.com/keboola/connection/pull/4925

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -6733,6 +6736,50 @@ For a more detailed guide on the file upload, see the corresponding part of the
...
+                   "name": "filePrepare",
+                   "isPermanent": true,
+                   "isSliced": false,
+                   "contentType": "text/csv",
+                   "tags": [
+                       "tag1",
+                       "tag2"
+                   ],
+                   "sizeBytes": 123456,
+                   "isPublic": true,
```
Comment (tomasfejfar) [inline apiary.apib:6758]:
```suggestion
                   "isPublic": false,
``` 
je to deprecated

File: tests/ClientTestCase.php [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 class ClientTestCase extends TestCase
 {
-    use \PHPUnitRetry\RetryTrait;
+//    use \PHPUnitRetry\RetryTrait;
```
Comment (tomasfejfar) [inline tests/ClientTestCase.php:15]: ❗

**Review by tomasfejfar — CHANGES_REQUESTED** [review]

File: tests/ClientTestCase.php (hunk 2 above)
Comment (zajca) [reply tests/ClientTestCase.php:15]: Kurňa zas :D

File: apiary.apib (hunk 1 above)
Comment (zajca) [reply apiary.apib]: to je ukázka response to je docela fuk ale upravil sem

File: tests/ClientTestCase.php (hunk 2 above)
Comment (zajca) [reply tests/ClientTestCase.php:15]: díx

Comment (tomasfejfar) [reply tests/ClientTestCase.php:15]: Furt ot tam je :D

Comment (zajca) [reply tests/ClientTestCase.php]: :D sorry dnes je už prakticky pátek :D

**Review by tomasfejfar — APPROVED** [review]

**Pull request merged**